CONTRIBUTING.md

```diff
@@ -14,14 +14,15 @@ Generally, we appreciate external contributions very much and would love to work
 
 ---
 
-<!-- MarkdownTOC autolink="true" -->
+<!-- TOC -->
```

Comment:
I think these headers were there for a reason for auto-generation of TOC. Maybe we need to settle on a tool to generate these, but for now I think it's best if we revert this small change.

Reply:
Sure, I guess that must be [the Sublime Plugin](https://packagecontrol.io/packages/MarkdownTOC) to generate the MarkdownTOC, as the header suggests. 

I don't have any preference, TBH. I favor any automated solution that makes it easier to traverse the documentation and requires minimal effort to maintain.  👍

Reply:
I've reverted this change, the conversation is marked as outdated, but not resolved, I don't know why. 
I'm resolving this conversation.